HISTORY.md

```diff
@@ -10,6 +10,7 @@
 ### New Features
 * ldb has a new feature, `list_live_files_metadata`, that shows the live SST files, as well as their LSM storage level and the column family they belong to.
 * The new BlobDB implementation now tracks the amount of garbage in each blob file in the MANIFEST.
+* Add support for Merge with base values (Put/Delete etc.) during compaction in Intergrated BlobDB.
```

Comment:
I think we could simply say that the integrated BlobDB now supports `Merge` (not just during compaction but in general).

Reply:
Sure. Thanks for checking that.